ovs_performance.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
Any reason why this was removed?  I think it doesn't belong with this commit.

Reply:
your right that was a mistake

Reply:
fixed